siliconcompiler/remote/client.py

```diff
@@ -152,6 +152,9 @@ def remote_preprocess(chip, steplist):
                                           args=(local_step, index, {}))
         run_task.start()
         run_task.join()
+        if run_task.exitcode != 0:
+            # A 'None' or nonzero value indicates that the Process target failed.
+            raise SiliconCompilerError("Could not start remote job: local setup task failed.")
```

Comment:
could you call `chip.error( fatal=True)` instead of directly raising.
Also the message might be a little more informative with:
"Could not start remote job: local setup task (<step><index>) failed."

Reply:
Sure, good idea.